Sources/PackageDescription/PackageDescription.swift

```diff
@@ -31,46 +31,56 @@ import Foundation
 /// following package manifest shows the initialization of a simple package
 /// object for the MyLibrary Swift package:
 ///
-///     // swift-tools-version:5.3
-///     import PackageDescription
+/// ```swift
+/// // swift-tools-version:5.3
+/// import PackageDescription
 ///
-///     let package = Package(
-///         name: "MyLibrary",
-///         platforms: [
-///             .macOS(.v10_15),
-///         ],
-///         products: [
-///             .library(name: "MyLibrary", targets: ["MyLibrary"]),
-///         ],
-///         dependencies: [
-///             .package(url: "https://url/of/another/package/named/Utility", from: "1.0.0"),
-///         ],
-///         targets: [
-///             .target(name: "MyLibrary", dependencies: ["Utility"]),
-///             .testTarget(name: "MyLibraryTests", dependencies: ["MyLibrary"]),
-///         ]
-///     )
+/// let package = Package(
+///     name: "MyLibrary",
+///     platforms: [
+///         .macOS(.v10_15),
+///     ],
+///     products: [
+///         .library(name: "MyLibrary", targets: ["MyLibrary"])
+///     ],
+///     dependencies: [
+///         .package(url: "https://url/of/another/package/named/utility", from: "1.0.0")
+///     ],
+///     targets: [
+///         .target(name: "MyLibrary", dependencies: ["Utility"]),
+///         .testTarget(name: "MyLibraryTests", dependencies: ["MyLibrary"])
+///     ]
+/// )
+/// ```
 ///
-/// The package manifest must begin with the string `// swift-tools-version:`,
-/// followed by a version number such as `// swift-tools-version:5.3`.
+/// The package manifest must begin with the string `// swift-tools-version:`
+/// followed by a version number specifier. The following code listing shows a
+/// few examples of valid declarations of the Swift tools version:
```

Comment:
This has rule has been relaxed since Swift 5.4. Manifest files for Swift ≥ 5.4 no long _must_ begin with "// swift-tools-version:" followed immediately by the version string. They can contain any combination of whitespace before and after "//" and after ":". 

Also the "swift-tools-version" part has always been case-insensitive, but the convention is all-lowercase.

Reply:
That sounds too loosy-goosey to me. I'd rather not see people do

Reply:
We can mention the relaxation around whitespace without having to go to the extreme in recommending the format as shown in your example. One of the benefits is for the user to have more control with comments' alignment and format in their entire project. For example, I can imagine some users might prefer to have all their comments indented by some whitespace, starting with an uppercase letter, or aligned with doc comments by using 1 more spaces or a tab after "//". Providing them with this information allows them to treat the Swift version specification as a regular comment in their source code, and format it as they prefer.